Craigslist/Craigslist.Selenium/Craigslist.Selenium.csproj

```diff
@@ -32,6 +32,12 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="PagedList">
```

Comment:
You don't need this package in Selenium. why did you add it here?
Did you see the changes before pull request?